reconcilers/reconcilers.go

```diff
@@ -10,6 +10,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	errors2 "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
I added this import to use erros2.Aggregate type for collecting all errors from managing all children. Feel free to suggest other approaches.